Content.Server/Objectives/Systems/KillPersonConditionSystem.cs

```diff
@@ -86,34 +90,49 @@ private void AssignRandomTarget(EntityUid uid, ObjectiveAssignedEvent args, Pred
 
         _target.SetTarget(uid, _random.Pick(selectedHumans), target);
     }
-
+    // DeltaV - start making people only die once from EE
     private float GetProgress(EntityUid target, bool requireDead)
     {
         // deleted or gibbed or something, counts as dead
         if (!TryComp<MindComponent>(target, out var mind) || mind.OwnedEntity == null)
+        {
+            if (!requireDead && !_wasKilled.Contains(target)) _wasKilled.Add(target);
```

Comment:
Sadly this doesn't quite work :(
This function only gets called when someone checks the progress (E.g opens the objective window). This means that if someone was killed, and then revived, and you never checked your objectives, it wouldn't count (See picture).

I think there are two ways to resolve this:
1.) Embrace it! Add like a button in the uplink or some kind of item or way of "contacting" the syndicate that to tell them the target is dead. If you do it like this, it would force you to actively know when your target died (randomly dying to a slime wouldn't count unless you knew exactly when it was happening.) which really plays into the theme I think.

2.) Add a component that keeps track the amount of times someone has died per round. This would solve the issue and would also allow you to add the number of times someone was killed for the round which is fun!

Both are valid solutions but I think 1.) is a little more roleplay focused while 2.) is a little more fun/silly focused. Maybe one of the curators would know?

(I killed and then revived them without checking the objective)
![Screenshot 2024-11-12 231435](https://github.com/user-attachments/assets/b8053d1f-34c6-4ce0-9b1c-54ecb48a18b7)